src/main/java/cpp/projects/ProjectList.java

```diff
@@ -129,22 +137,44 @@ public void addDeadline(String title, String deadline) {
         System.out.println(Constants.SEPARATOR);
     }
 
+    /**
+<<<<<<< HEAD
+     * Gets the total number of projects in the list (for test use).
+     *
+     * @return total number of projects in the list
+     */
+
+    public int getProjectNo() {
+        return projectList.size();
+    }
+
+    private int findProjectIndex(String name) {
+        int index = -1;
+        int count = 0;
+
+        for (Project project : projectList) {
+            if (project.getTitle().equalsIgnoreCase(name)) { //find the index of the project
+                index = count;
+                break;
```

Comment:
Not that important, but you can change this to simply return count and return -1 otherwise. `index` is technically not that necessary

Reply:
Noted. Thanks for your suggestion!